tests/test_tuning_TSINIT_NEXTSTEP.cpp

```diff
@@ -5,7 +5,7 @@
 #include <iostream>
 
 #define BOOST_TEST_MODULE TestTuningTSINIT
-#include <boost/test/unit_test.hpp>
+#include <boost/test/included/unit_test.hpp>
```

Comment:
On my system there is no boost/test/included directory.

Can you please try whether it works for when adding 
```
#include <config.h>
```

at the top instead? That file should have
```
#define HAVE_DYNAMIC_BOOST_TEST 1
```
in it which probably does the trick.